frr-metrics/collector/bgp_test.go

```diff
@@ -16,37 +16,91 @@ var (
 	# HELP metallb_bgp_announced_prefixes_total Number of prefixes currently being advertised on the BGP session
 	# TYPE metallb_bgp_announced_prefixes_total gauge
 	metallb_bgp_announced_prefixes_total{peer="{{ .NeighborIP }}"} {{ .AnnouncedPrefixes }}
+	# HELP metallb_bgp_keepalives_received Number of BGP keepalive messages received
```

Comment:
nit: can you change the commit subject with the part you are adding the tests for?

Reply:
done